javascripts/discourse/initializers/dc-homepage.js.es6

```diff
@@ -1,9 +1,13 @@
+import { setDefaultHomepage } from "discourse/lib/utilities";
 import { withPluginApi } from "discourse/lib/plugin-api";
 
 export default {
   name: "dc-homepage",
   initialize() {
     withPluginApi("0.8", api => {
+      // Define arbitrary hompage url
+      setDefaultHomepage("/latest");
```

Comment:
@duranmla can you explain a bit why are we doing this? Do you know if we have this available in the plugin API?

Reply:
I was taking a look to the core solutions for this but there is no way to force homepage to be `/latest` _(at least I wasn't able to find it)_ instead I found this plugin created by one of the Discourse contributors 

_I am not able to get back into the topic where he mention about this and the whole debate happens_